### DISCUSSION TITLE
Security Overview reporting wrong number of open alerts

### DISCUSSION THREAD

**Discussion by prakashsivakumars** [community/community#83053] [Code Security] [answered]
### Select Topic Area

Show & Tell

### Body

I'm not sure how the open alerts are calculated in Security Overview beta. 
This is the numbers in the actual repository
<img width="921" alt="image" src="https://github.com/community/community/assets/94060004/78814ffa-92e1-401f-bae4-fe93082355ad">
This is the numbers in the security overview:
<img width="569" alt="image" src="https://github.com/community/community/assets/94060004/186f1b8c-9643-469f-ad8e-a8273e23cb24">

**Comment by just-joshing** [comment]
Hello @prakashsivakumars! Sorry for the delayed response.
Are you still seeing this discrepancy?

**Comment by prakashsivakumars** [comment]
Hi!! Yes,, Im still facing that issue. Please let me know if you need any more info.

Thank you,
Prakash

Sent via the Samsung Galaxy S22 5G, an AT&T 5G smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by itsbigshawn** [comment]
Hello @prakashsivakumars! 👋  Would you mind clarifying which number in the Overview Dashboard you were comparing against? As well as any filters that were applied at the time, if any?

**Reply by prakashsivakumars** [reply]
Any repository --> Security --> Code Scanning Numbers(on the left side) Vs Security Overview(Beta) -- > Apply Filter for Same Repository --> Scroll Down to the table at the bottom and look that the number for that repository

**Reply by itsbigshawn** [reply]
Got it, so you're comparing Code Scanning numbers specifically. 

> Apply Filter for Same Repository

To clarify, did you also apply `tool:CodeQL` on top of the repository filter? I ask because otherwise the dashboard would still show the alert counts from all security tools.

**Reply by prakashsivakumars** [reply]
Yes.

**Comment by itsbigshawn** [comment, marked as answer]
Hi @prakashsivakumars! 👋 

Apologies for the delay, but the issue should be fixed now. Please don't hesitate to reach out again if you run into another issue 🙂

**Reply by prakashsivakumars** [reply]
Thank you! Its working and I see numbers in both places.
